core/src/mast/merger/mod.rs

```diff
@@ -163,6 +167,17 @@ impl MastForestMerger {
         Ok(())
     }
 
+    fn merge_advice_map(&mut self, other_forest: &MastForest) -> Result<(), MastForestError> {
+        for (key, value) in other_forest.advice_map.clone().into_iter() {
```

Comment:
```suggestion
        for (key, value) in other_forest.advice_map.iter() {
```

Can you implement `winter_crypto::utils::collections::KvMap` for `AdviceMap`, which will give us the `iter()` method? `KvMap` is already implemented for `BTreeMap`, so it's just a matter of forwarding all calls

Reply:
Sure. Done.